DEVELOPER_GUIDE.md

```diff
@@ -2,74 +2,71 @@
+- This is F#: although F#+ contains some abstractions inspired in libraries used in other languages, those concepts are translated to F# standards and eventually adapted to fit better in F#.
 
 ### Extensions
 
 This library defines many extensions for different types.
 Some of these functions are conceptually connected, in that they represent an abstraction and sometimes they have indeed a generic version of the function which doesn't require typing the type as module prefix.
 Note that in these cases, normally the names are the same but this is not a rule, there are cases where the name differs to avoid collisions or confusion with other functions, for example:
 
- - `map` is a generic function, and we have many types implementing it, but for dictionaries this correspond to `mapValues`
- - `min` is a non-generic function operating in collections, but its generic counterpart is `minimum` to avoid collision with the built-in `min` function (minimum between two values).
- 
-So, what this means is, we're not providing generic functions based on names, although in many cases names are the same, we need to take into account:
- 
- - F# core has some inconsistencies as it's not a type classes based library. A typeclasses based library is not necessarily something that implements a trick for typeclasses, but something that is designed as if we had support for them, I mean capturing some generic concepts and making them clear in the names chosen for every functions.
+- `map` is a generic function, and F#+ has many types implementing it, but for dictionaries this correspond to `mapValues`
+- `min` is a non-generic function operating in collections, but its generic counterpart is `minimum` to avoid collision with the built-in `min` function (minimum between two values).
 
- - But here we're not attempting to fix F# core, we want to build on top in a non intrusive way, and try to re-use all the consistent concepts, idioms and de-facto naming conventions from F# core as much as possible without increasing the inconsistency level already there. Keep in mind, in F#+ **we try to expand FSharp.Core functionalities, but we are trying not to be intrusive**, as stated before: F#+ builds upon FSharp, using generic programming techniques to help avoid boiler plate code. However, by naming conventions and signatures it can be seen to 'enhance' rather than 'replace' existing patterns as much as possible.
+So, what this means is, F#+ doesn’t provide generic functions based on names, although in many cases names are the same, we need to take into account:
+
+1. F# core has some inconsistencies as it's not a type classes based library. A `typeclasses` based library is not necessarily something that implements a trick for `typeclasses`, but something that is designed as if we had support for them, it means capturing some generic concepts and making them clear in the names chosen for every functions.
```

Comment:
I guess these instances of "typeclasses" should be unquoted too, as in this context, the word just refers to the actual concept of type classes. We also need a space between "type" and "classes", and

> type-class based library

would sound more natural.